### PR TITLE
Docs #25468 : add code examples for drop and dropWhile Operators

### PR DESCRIPTION
**Pull request by rvacaru**
Documentation snippets for drop simple operators.
References #25468

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Comment by raboof** [comment]
OK TO TEST

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Review by johanandren — COMMENTED** [review]
Good, one suggestion on the dropWhile

**Comment by rvacaru** [comment]
I signed the Lightbend CLA

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/dropWhile.md [hunk 1]
```diff
@@ -16,6 +16,16 @@ Drop elements as long as a predicate function return true for the element
 
 Drop elements as long as a predicate function return true for the element
 
+## Example
+
+Given a `Source` of ordered numbers we can drop all the negative ones with the `dropWhile` operator:
```
Comment (ennru) [inline akka-docs/src/main/paradox/stream/operators/Source-or-Flow/dropWhile.md]: Please describe the distinction to `filter`: `dropWhile` will emit all elements after the expression returns false for an element.

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/dropWhile.md (hunk 1 above)
Comment (rvacaru) [reply akka-docs/src/main/paradox/stream/operators/Source-or-Flow/dropWhile.md]: Added additional explanations

**Comment by lightbend-cla-validator** [comment]
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

**Review by ennru — APPROVED** [review]
LGTM.

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Pull request merged**